adventure.py

```diff
@@ -0,0 +1,260 @@
+#money_left()
+def money_left(money):
+    print("\nYou now have " + str(money) + " microbucks left.")
+    print("What would you like to do with it? (1 or 2)")
+    print("1) Save the rest up to buy a house later")
+    print("2) Buy a hoverboard")
+
+    answer = input(">")
+
+    if answer == "1":
+        print("Good idea, this will be a good investment.")
+
+    elif answer == "2":
+        money -= 50
+        print(" You decided to buy the cheapest hoverboard you can find.")
+        print("You buy a hoverboard for 50 microbucks.  It says it can hold 2 people.")
+        print("You have " + str(money) + " microbucks left.")
```

Comment:
you should use fstrings

Reply:
```suggestion
        print(f"You have {money} microbucks left.")
```

Reply:
Good idea I have no idea how to do that but I do now.